docs/cloud/tcld/login.md

```diff
@@ -0,0 +1,20 @@
+---
+id: login
+title: tcld login
+sidebar_label: login
+description: How to log in to Temporal Cloud using tcld.
+tags:
+  - reference
+  - tcld
+---
+
+The `tcld login` command logs in a user to Temporal Cloud.
+
+Follow instructions in the browser to log in to your Temporal account.
+After successful authentication, tcld creates a file in `~/.config/tcld` that contains the JSON web token (JWT) that tcld uses to authenticate requests made to the SaaS API.
```

Comment:
just curious is it necessary to include info about the config dir here?

Reply:
Also this implementation will probably change over time making it hard to keep this section in the docs up to date--i might consider removing this.

Reply:
I'm good with removing it.